### PR TITLE
fix: build skopeo binary

### PR DESCRIPTION
**Pull request by parthasl**
## Context

Build Skopeo binary from source and upload to Screwdriver bintray repository

## References

https://github.com/containers/skopeo/blob/master/install.md#building-from-source
https://bintray.com/screwdrivercd/screwdrivercd/skopeo/1.0.0#files

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Copyright 2016 Yahoo Inc.
```
Comment (jithine) [inline LICENSE]: this year should be 2021. it's the year this file is added

Comment (parthasl) [reply LICENSE]: updated year should it be VZM or Yahoo Inc. ?

Comment (jithine) [reply LICENSE]: `Verizon Media`

**Review by jithine — DISMISSED** [review]

**Review by jithine — APPROVED** [review]

**Comment by pritamstyz4ever** [comment]
closing this, created https://github.com/screwdriver-cd/sd-packages/pull/2

**Pull request closed without merging**